scene/3d/occluder_instance_3d.cpp

```diff
@@ -314,6 +317,30 @@ OccluderInstance3D::BakeError OccluderInstance3D::bake(Node *p_from_node, String
 	return BAKE_ERROR_OK;
 }
 
+TypedArray<String> OccluderInstance3D::get_configuration_warnings() const {
+	TypedArray<String> warnings = Node::get_configuration_warnings();
+
+	if (!bool(GLOBAL_GET("rendering/occlusion_culling/use_occlusion_culling"))) {
+		warnings.push_back(TTR("Occlusion culling is disabled in the Project Settings, which means occlusion culling won't be performed in the root viewport.\nTo resolve this, open the Project Settings and enable Rendering > Occlusion Culling > Use Occlusion Culling."));
+	}
+
+	if (bake_mask <= 4'293'918'720) {
```

Comment:
Isn't it simpler and less hardcoded to check `bake_mask == 0`?

Reply:
I did that initially, but it doesn't work if you merely uncheck the 20 layer checkboxes available in the editor. The reason is that the bake mask is initialized to `INT32_MAX`, which means *all* layers are enabled by default – including the 12 ones you currently can't toggle from the editor.

Reply:
That seems pretty hacky to me to workaround an editor limitation. If users uncheck the 20 visible layers, then there's still a valid `bake_mask` and the node is not misconfigured, is it?

Reply:
> If users uncheck the 20 visible layers, then there's still a valid bake_mask and the node is not misconfigured, is it?

Yes, but you'd have to use a tool script to set meshes' masks to enable layers that can't be enabled in the editor (since only the first layer is enabled by default on VisualInstance3D). This is very unlikely.

Reply:
Still, I'm not sure what this particular warning is trying to achieve. Warning users only if they decided to untick all 20 visible bits one by one doesn't seem like a use case that warrants hacking around something editor specific (which we might forget to un-hack once the editor usability issue is resolved).

IMO if there should be a warning here, it's for 0 (set from script or by editing `tscn` or custom node that has a different default value). And we should speed up resolution of https://github.com/godotengine/godot-proposals/issues/2770 (which is increasingly becoming a bug more than a feature proposal).